### PR TITLE
Add python-pyjwt-pip to python.yaml

### PR DESCRIPTION
**Pull request by be-ez**
[PyJWT ](https://pyjwt.readthedocs.io/en/latest/) is commonly used to encode and decode JSON Web Tokens. PyJWT is also required for authentication with Google IoT Core.

### CODE REVIEW

**Review by tfoote — CHANGES_REQUESTED** [review]
It looks like python-jwt has mature packages in the apt repositories we should use them instead of the pip packages. 

Ubuntu: https://packages.ubuntu.com/xenial/python/python-jwt
Debian: https://packages.debian.org/search?keywords=python-jwt
Fedora: https://apps.fedoraproject.org/packages/python-jwt
Gentoo: https://packages.gentoo.org/packages/dev-python/pyjwt

**Comment by be-ez** [comment]
@tfoote I'm usually inclined to use the pip versions of pyjwt because the repository version varies between distros. 

Current Version: 1.7.1 (December 2018) 
Ubuntu: 1.3.0 ( May 2015) 
Debian: 1.4.2 ( August 2016 ) 
Fedora: 1.6.4 ( May 2018 ) 
Gentoo: 1.5.3 ( Sept 2017 ) 


I've made the changes per your request.

File: rosdep/python.yaml [hunk 1]
```diff
@@ -2780,6 +2780,11 @@ python-pyinotify:
   fedora: [python-inotify]
   gentoo: [dev-python/pyinotify]
   ubuntu: [python-pyinotify]
+python-pyjwt-pip:
```
Comment (cottsay) [inline rosdep/python.yaml]: Since the package no longer uses pip, the suffix should be dropped.
```suggestion
python-jwt:
```

**Review by cottsay — CHANGES_REQUESTED** [review]

**Review by tfoote — APPROVED** [review]

**Review by cottsay — APPROVED** [review]

**Comment by tfoote** [comment]
> I'm usually inclined to use the pip versions of pyjwt because the repository version varies between distros.

Indeed it's a little bit more effort to make sure that you can support the versions packaged and the apt based versions are usually behind the pip version. But there's a lot more challenges with the pip version for the larger system integration. The apt version has been tested and qa'd against all the other packages on the system. Whereas the pip version will just roll ahead at the developers desired pace and can often break things already installed on the system. The pip version also will not be upgraded when doing standard system updates, and we've seen it many times that a new version is installed from pip, but after upgrades from apt, you still get an old behavior because the pip version is still shadowing it and it's not visible to the user why.

**Pull request merged**